lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ContinuousRankedProbabilityScoreMinimisers.py

```diff
@@ -302,103 +304,132 @@ def test_basic_mean_predictor_bad_value(self):
+            self.historic_wind_speed_forecast_cube.collapsed(
+                "realization", iris.analysis.VARIANCE))
+
+        self.initial_guess_for_mean = np.array([0, 1, 0, 1], dtype=np.float32)
+        self.initial_guess_for_realization = (
+            np.array([0, 1, 0, np.sqrt(1/3.), np.sqrt(1/3.), np.sqrt(1/3.)],
+                     dtype=np.float32))
+
+
+class Test_crps_minimiser_wrapper_gaussian_distribution(SetupInputs):
 
     """
     Test minimising the CRPS for a normal distribution.
     Either the ensemble mean or the individual ensemble realizations are used
     as the predictors.
     """
+    def setUp(self):
+        """Set up expected output."""
+        super().setUp()
+        self.expected_mean_coefficients = (
+            [-0.000236, 0.797574, 0.000423, 0.997329])
+        self.expected_realizations_coefficients = (
+            [0., 1.05, 0., 0.5774, 0.5774, 0.5774])
+
     @ManageWarnings(
         ignored_messages=["Collapsing a non-contiguous coordinate.",
-                          "Minimisation did not result in convergence"])
-    def test_basic_normal_mean_predictor(self):
+                          "Minimisation did not result in convergence",
+                          "divide by zero encountered in"],
+        warning_types=[UserWarning, UserWarning, RuntimeWarning])
+    def test_basic_mean_predictor(self):
         """
         Test that the plugin returns a numpy float value.
         The ensemble mean is the predictor.
         """
-        expected = [0.303724, -0.019689, 0.011536, 1.00955]
-        initial_guess = [5, 1, 0, 1]
-        initial_guess = np.array(initial_guess, dtype=np.float32)
-        cube = set_up_temperature_cube()
-
-        forecast_predictor = cube.collapsed("realization", iris.analysis.MEAN)
-        forecast_variance = cube.collapsed(
-            "realization", iris.analysis.VARIANCE)
-        truth = cube.collapsed("realization", iris.analysis.MAX)
-
         predictor_of_mean_flag = "mean"
-
-        plugin = Plugin()
         distribution = "gaussian"
+        plugin = Plugin()
         result = plugin.crps_minimiser_wrapper(
-            initial_guess, forecast_predictor, truth, forecast_variance,
+            self.initial_guess_for_mean,
+            self.temperature_forecast_predictor_mean,
+            self.temperature_truth_cube,  self.temperature_forecast_variance,
             predictor_of_mean_flag, distribution)
         self.assertIsInstance(result, np.ndarray)
         self.assertEqual(result.dtype, np.float32)
-        self.assertArrayAlmostEqual(result, expected)
+        self.assertArrayAlmostEqual(
+            result, self.expected_mean_coefficients)
 
     @ManageWarnings(
         ignored_messages=["Collapsing a non-contiguous coordinate.",
-                          "Minimisation did not result in convergence"])
-    def test_basic_normal_realizations_predictor(self):
+                          "Minimisation did not result in convergence",
```

Comment:
Why are we ignoring this?  It seems pretty relevant in the context of recent problems.

Reply:
This still worries me, but again, probably OK to leave this for future plans.